### ISSUE TITLE
HA Logs tell me to open an issue

### ISSUE THREAD

**Issue by macjustice** [bodyscape/cielo_home#55]
Just updated HA and noticed this in the logs:

```
2024-03-11 18:28:46.360 WARNING (MainThread) [homeassistant.components.climate] Entity None (<class 'custom_components.cielo_home.climate.CieloHomeThermostat'>) implements HVACMode(s): off, cool, dry, fan_only, auto, heat and therefore implicitly supports the turn_on/turn_off methods without setting the proper ClimateEntityFeature. Please create a bug report at https://github.com/bodyscape/cielo_home/issues
```

Haven't noticed any issues, if I'm reading that correctly I'm guessing maybe they changed the spec for ClimateEntity?

Home Assistant Container
Core 2024.3.0
Frontend 20240306.0

**Comment by bodyscape** [comment]
I'll check on it.

**Comment by bodyscape** [comment]
fix in 1.7.6
